src/component/user/user.module.scss

```diff
@@ -0,0 +1,61 @@
+
+$dark-brown: rgba(46 30 20);
```

Comment:
🤔 can you add these colors into `/styles/variables.scss` and then here import `styles.scss` so that you can use those colors

Reply:
I changed the hover colors to be a little lighter.  Did you want me to use the other colors?  I was trying to make it look pretty light.

Reply:
I might have forgotten to rename it to hover on the colors to make them different.  I could also do that.

Reply:
As long as it is added into variables so that others can use it would be fine.

Reply:
I think I did what you wanted.  Let me know if there is anything else that is needed